core/domain/story_domain.py

```diff
@@ -776,7 +776,10 @@ def validate(self):
         if not isinstance(self.notes, str):
             raise utils.ValidationError(
                 'Expected notes to be a string, received %s' % self.notes)
-
+        if len(self.notes) > 5000:
+            raise utils.ValidationError(
+                'Expected notes length to be less than 5000,'
+                ' but it is %s long' % len(self.notes))
```

Comment:
This is not in line with the check above, should it be less than 5000 or less or equal to 5000?